### DISCUSSION TITLE
Can't hide iframe's background while on dark mode

### DISCUSSION THREAD

**Discussion by pkrakas** [tailwindlabs/tailwindcss#12057] [Help] [answered]
Hello,

I am not able to find what's causing background to be white. Everything is transparent and white background still shows up. However this doesn't happen on light mode, when background is properly transparent.

![dark_mode](https://github.com/tailwindlabs/tailwindcss/assets/18681314/1048bb8f-9ff1-4921-9e4b-2532fb0c9bcf)
![light_mode](https://github.com/tailwindlabs/tailwindcss/assets/18681314/77815cc3-7af5-447b-96b1-06b8865880e0)

That's weird, I found that this happens only if dark mode is the default one on page load. If I open webpage with light mode initially set, then after switching to dark, everything is OK. On the otherside, if I start with dark mode, white square appears in the background of iframe.

![150259](https://github.com/tailwindlabs/tailwindcss/assets/18681314/aa96c36e-297b-448e-b510-20e611bbc7fd)
![150347](https://github.com/tailwindlabs/tailwindcss/assets/18681314/7af50cf7-5681-447c-b7d4-ddbceacc9354)

**Comment by wongjn** [comment]
Consider providing a reproduction please, so that we can better assist in troubleshooting.

**Reply by pkrakas** [reply, marked as answer]
Found the solution. That's because Web3Modal does this:
![223656](https://github.com/tailwindlabs/tailwindcss/assets/18681314/7774532c-13e0-4d70-9ef3-311dbc2190ea)

After changing color-scheme to 'normal', everything works as it should
